plugins/main/windows/overlay/sample.conf

```diff
@@ -0,0 +1,36 @@
+{
+    "cniVersion":"0.2.0",
+    "name":"vxlan0",
+    "type":"flannel",
+    "delegate":{
+        "type":"win-overlay",
+        "dns":{
+            "nameservers":[
+                "11.0.0.10"
+            ],
+            "search":[
+                "svc.cluster.local"
+            ]
+        },
+        "additionalArgs":[
+            {
+                "name":"EndpointPolicy",

```

Comment:
This should probably be "kind" instead of "name"

Reply:
All things on `additionalArgs` will be set into `HNSEndpoint`.`Policies` field directly.